public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -169,43 +218,34 @@ const useFilteredSilences = (silences: Silence[], expired = false) => {
 };
 
 const getStyles = (theme: GrafanaTheme2) => ({
-  topButtonContainer: css`
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
-  `,
-  addNewSilence: css`
-    margin: ${theme.spacing(2, 0)};
-  `,
-  callout: css`
-    background-color: ${theme.colors.background.secondary};
-    border-top: 3px solid ${theme.colors.info.border};
-    border-radius: ${theme.shape.radius.default};
-    height: 62px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
+  addNewSilence: css({
```

Comment:
This seems to be unused

Reply:
Thanks - that's the thing I don't like about the approach for styles, it doesn't get caught by linting 😁